javascript/src/main/components/ChannelMessages/MessageList.js

```diff
@@ -29,15 +29,15 @@ const messageContents = (text, users) => {
 }
 
 
-export default ({ messages, users}) => {
+export default ({ messages, users = [] }) => {
     const columns = [{
         isDummyField: true,
-        formatter: (cell, row) => userName(row.user,users),
+        formatter: (cell, row) => userName(row.user,users||[]),
```

Comment:
Should be unnecessary due to above default.
```suggestion
        formatter: (cell, row) => userName(row.user, users),
```

Reply:
done